content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -239,9 +241,23 @@ message ContainerDevices {
 }
 ```
 
+### `GetAllocatableResources` gRPC endpoint
```

Comment:
I'd set the fragment identifier explicitly:
```suggestion
###  ### `GetAllocatableResources` gRPC endpoint {#grpc-endpoint-getallocatableresources}
```

Reply:
Ack